util/config/fixpath.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import os
+from sys import argv
+
+
+def main(path, sep=":"):
+    """
+    Removes path components that begin with $CHPL_HOME, to reduce
+    $PATH & $MANPATH pollution
+
+    :path: $PATH or $MANPATH
+    :sep: Expected path separator
+    :returns: new path with $CHPL_HOME components stripped
+    """
+    chpl_home = os.getenv("CHPL_HOME")
+    newpath = [p for p in path.split(sep) if chpl_home not in p]
```

Comment:
The other issue with spaces in the path shows up here. If my path looks like: `/bin /usr/bin /home/Kyle\ B/$CHPL_HOME` PATH is going to get corrupted.